src/__tests__/user.spec.js

```diff
@@ -0,0 +1,184 @@
+import chai from 'chai';
```

Comment:
move test into __tests__/routes/profiles/profileSettings.js

Reply:
okay, instead of the profile, it should be inside users i.e. `tests/routes/users/profileSettings`

Reply:
Oh yes, that is better since it is served on the users route